dev/src/DocFx/Node/XrefTrait.php

```diff
@@ -179,14 +188,19 @@ private function replaceUidWithLink(string $uid, string $name = null): string
             case 0 === strpos($uid, '\Google\Type\\'):
                 $extLinkRoot = 'https://googleapis.github.io/common-protos-php#';
                 break;
+            case 0 === strpos($uid, '\GuzzleHttp\Promise\PromiseInterface'):
```

Comment:
nit: let's use the "new" PHP method `str_starts_with` to make all these more readable
```suggestion
            case 0 === strpos($uid, '\GuzzleHttp\Promise\PromiseInterface'):
```

https://www.php.net/manual/en/function.str-starts-with.php